src/cljs/org/broadinstitute/firecloud_ui/paths.cljs

```diff
@@ -45,5 +45,8 @@
 (defn submission-details [workspace-id submission-id]
   (str (submissions-list workspace-id) "/" submission-id))
 
+(defn submission-kill [workspace-id submission-id]
```

Comment:
To be more consistent with the other path method names and more closely align to the named functionality, this should be renamed to `get-abort-submission-path`